src/workerd/api/node/node.h

```diff
@@ -54,7 +55,8 @@ class CompatibilityFlags: public jsg::Object {
   V(ZlibUtil, "node-internal:zlib")                                                                \
   V(UrlUtil, "node-internal:url")                                                                  \
   V(DnsUtil, "node-internal:dns")                                                                  \
-  V(TimersUtil, "node-internal:timers")
```

Comment:
For now, the new internal module should be guarded by the `--experimental` flag.